content/en/security/cloud_siem/investigate_security_signals.md

```diff
@@ -50,36 +50,93 @@ To view your signals by MITRE ATT&CK Tactic and Technique:
+    - You can also see specific information like username or network IP. In **Rule Details**, click the funnel icon to create a suppression rule or add the information to an existing suppression. See [Create suppression rule][11] for more details.
+1. In the **Next Steps** section:   
+  a. Under **Triage**, click the dropdown to change the triage status of the signal. The default status is `OPEN`.
+      - `Open`: Datadog Security triggered a detection based on a rule, and the resulting signal is not yet resolved.
+      - `Under Review`: During an active investigation, change the triage status to `Under Review`. From the `Under Review` state, you can move the status to `Archived` or `Open` as needed.
+      - `Archived`: When the detection that caused the signal has been resolved, update the status to `Archived`. When a signal is archived, you can give a reason and description for future reference. If an archived issue resurfaces, or if further investigation is necessary, the status can be changed back to `Open`. All signals are locked 30 days after they have been created.</ul>
+  b. Click **Assign Signal** to assign a signal to yourself or another Datadog user.
+  c. Under **Take Action**, you can create a case, declare an incident, edit suppressions, or run workflows. Creating a case automatically assigns the signal to you and sets the triage status to `Under Review`.
+
+{{< img src="security/security_monitoring/investigate_security_signals/signal_side_panel.png" alt="The signal side panel of a compromised AWS IAM user access key showing two IP addresses and their locations" style="width:90%;" >}}
+
+### Triage multiple signals
 
-Use bulk actions to triage multiple signals. To use bulk actions, first search and filter your signals in the Signal Explorer, then:
+Use bulk actions to triage multiple signals. To use bulk actions, first search and filter your signals in the Signals Explorer, then:
 
-1. Click on the checkbox to the left of the signals that you want to take a bulk action on. To select all signals in the Signal Explorer list, select the checkbox next to the **Status** column header.
+1. Click on the checkbox to the left of the signals that you want to take a bulk action on. To select all signals in the Signals Explorer list, select the checkbox next to the **Status** column header.
 1. Click on the **Bulk Actions** dropdown menu above the signals table and select the action you want to take.
 
 **Note**: The Signals Explorer stops dynamically updating when performing a bulk action.
 
-{{< img src="security/security_monitoring/investigate_security_signals/bulk_actions.png" alt="The Signal Explorer showing the bulk action option" style="width:45%;" >}}
+{{< img src="security/security_monitoring/investigate_security_signals/bulk_actions2.png" alt="The Signal Explorers showing the bulk action option" style="width:55%;" >}}
+
+### Run Workflow automation
```

Comment:
The product is called Workflow Automation. If this heading refers to the product, it should say "Run Workflow Automation." If you're talking about a workflow as a concept instead of the product name, it should be "Run workflow automation"

Reply:
Thanks @urseberry for pointing this out! I was following what's in the UI, which is calling it Datadog Workflows. I'll let the PM know that it's actually Workflow Automation. For the docs, I updated the header and committed your suggestions so it's referred to as Workflow Automation.